snet_cli/identity.py

```diff
@@ -117,18 +117,18 @@ def __init__(self, w3, index):
         self.client = TrezorClient(HidTransport.enumerate()[0])
         self.index = index
         self.address = self.w3.toChecksumAddress(
-            "0x" + bytes(self.client.ethereum_get_address([44 + bip32utils.BIP32_HARDEN,
-                                                           60 + bip32utils.BIP32_HARDEN,
-                                                           bip32utils.BIP32_HARDEN, 0,
+            "0x" + bytes(self.client.ethereum_get_address([44 + 0x80000000,
```

Comment:
I would propose introduce the constant for this number.

Reply:
My first thought was to use it from `pycoin` but it seems to use the hex "directly" too.
But I agree with you, changing...